### DISCUSSION TITLE
Landing Page Top Navbar

### DISCUSSION THREAD

**Discussion by PedroGarcia242** [colinwilson/lotusdocs#168] [Q&A]
I'm currently having trouble with adding buttons to the default landing page's navbar. I haven't even been able to replicate the navbar in the default https://lotusdocs.dev/ website, by adding the Docs, Community, Github and X links.
How is this done? Which file do I need to edit?
Thanks in advance.

**Comment by smp1lembang** [comment]
same, here too

**Comment by rkvsraman** [comment]
I am looking for this too.

**Comment by Jukim2** [comment]
it you see header.html, which is in the themes>lotusdocslayouts>partilas>, you can see some navigation and social links parts.

and i f you see the hugo.toml of the themes>loctusdocs>exampleSite, there is some setting about nav and social links.
